mcs/class/System.Drawing/System.Drawing/gdipFunctions.cs

```diff
@@ -979,22 +988,22 @@ static internal void CheckStatus (Status status)
 		internal static extern Status GdipBitmapSetPixel (IntPtr bmp, int x, int y, int argb);
 
 		// Image functions
-		[DllImport(GdiPlus, CharSet=CharSet.Auto)]
+		[DllImport(GdiPlus, CharSet=GdiPlusCharSet)]
 		internal static extern Status GdipLoadImageFromFile ( [MarshalAs(UnmanagedType.LPWStr)] string filename, out IntPtr image );
```

Comment:
I checked, for `LOGFONT` there's a `LOGFONTA` and `LOGFONTW` both in [libgdiplus](https://github.com/mono/libgdiplus/blob/1cbd58ef57c208a8517a772f3c8416169d7dfd9b/src/win32structs.h#L151) and [Windows](https://msdn.microsoft.com/en-us/library/windows/desktop/dd145037(v=vs.85).aspx).

So we should be able to just update the` LOGFONT` struct to use Unicode and have all the P/Invoke function declarations use Unicode as well.

I'll make the change and push an update.